test/exercism/use_cases/attempt_test.rb

```diff
@@ -12,7 +12,9 @@ def setup
   def test_validity
     Xapi.stub(:exists?, true) do
       refute Attempt.new(user, 'CODE', 'two.py').valid?
-      assert Attempt.new(user, 'CODE', 'python/two/two.py').valid?
+      solution = {'ruby/two/two.py' => 'CODE'}
+      opts = {track: 'ruby', slug: 'two'}
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.